app/tests/spec/models/verification/reset-password.js

```diff
@@ -0,0 +1,94 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+define([
+  'chai',
+  'sinon',
+  'lib/constants',
+  'models/verification/reset-password',
+  '../../../lib/helpers'
+], function (chai, sinon, Constants, Model, TestHelpers) {
+  var assert = chai.assert;
+
+  describe('models/verification/reset-password', function () {
```

Comment:
Do we need to add an assertion / test to make sure that the `EXPIRED_VERIFICATION_LINK` error works?